Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp

```diff
@@ -344,7 +340,7 @@ void CAntiAim::Run(CUserCmd* pCmd, bool* pSendPacket) {
 			   }
 			   else 
 			   {
-				  *pSendPacket = bSendReal = false;
+				  *pSendPacket == bSendReal == false;
```

Comment:
This line is useless, if you're not using the result of this expression

Reply:
when did i ask 😭